qubes/mgmt.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, app, src, method, dest, arg):
         #: name of the method
         self.method = method.decode('ascii')
 
+        #: callback for sending events if applicable
+        self.send_event = send_event
+
+        #: is this operation cancellable?
+        self.cancellable = False
+
```

Comment:
Yeah, and what happens if client closes connection for some method not marked as cancellable?